### PR TITLE
Using system command to get stdout,stderr from backup script

### PR DESCRIPTION
**Pull request by cweiland**

### CODE REVIEW

File: cron/centreon-backup.pl [hunk 1]
```diff
@@ -374,7 +375,8 @@ ()
         my @partialBackupDays = split(/,/, $BACKUP_DATABASE_PARTIAL);
         if ( grep $_ == $dayOfWeek, @partialBackupDays ) {
             print "Dumping Db with LVM snapshot (partial)\n";
-            `$centreon_config->{CentreonDir}cron/centreon-backup-mysql.sh -b $TEMP_DB_DIR -d $today -p`;
+	    $lvmBackupCmd.=" -p";
+            system($lvmBackupCmd);;
```
Comment (kduret) [inline cron/centreon-backup.pl]: two ";"

File: cron/centreon-backup.pl [hunk 2]
```diff
@@ -374,7 +375,8 @@ ()
         my @partialBackupDays = split(/,/, $BACKUP_DATABASE_PARTIAL);
         if ( grep $_ == $dayOfWeek, @partialBackupDays ) {
             print "Dumping Db with LVM snapshot (partial)\n";
-            `$centreon_config->{CentreonDir}cron/centreon-backup-mysql.sh -b $TEMP_DB_DIR -d $today -p`;
+	    $lvmBackupCmd.=" -p";
```
Comment (kduret) [inline cron/centreon-backup.pl]: bad indentation

File: cron/centreon-backup.pl [hunk 3]
```diff
@@ -360,12 +360,13 @@ ()
     my @localtime = localtime(time);
     my $dayOfWeek = @localtime[6];
     my @fullBackupDays = split(/,/, $BACKUP_DATABASE_FULL);
+    my $lvmBackupCmd="$centreon_config->{CentreonDir}cron/centreon-backup-mysql.sh -b $TEMP_DB_DIR -d $today";
```
Comment (kduret) [inline cron/centreon-backup.pl]: need space around "="

File: cron/centreon-backup.pl (hunk 2 above)
Comment (cweiland) [reply cron/centreon-backup.pl]: Corrected on 191ebdd

File: cron/centreon-backup.pl (hunk 1 above)
Comment (cweiland) [reply cron/centreon-backup.pl]: Corrected on 191ebdd

File: cron/centreon-backup.pl (hunk 3 above)
Comment (cweiland) [reply cron/centreon-backup.pl]: Corrected on 191ebdd

**Comment by garnier-quentin** [comment]
Sorry. I don't understand your PR. You change the backticks with system function.
```
system function:  it returns only the command return code, not the STDOUT and STDERR
backticks: return value is STDOUT
```

And the PR title: "Using system command to get stdout,stderr from backup script". You do the opposite in fact.

**Pull request closed without merging**